src/components/Calendar/index.js

```diff
@@ -162,4 +173,5 @@ Calendar.defaultProps = {
     className: undefined,
     style: undefined,
     id: undefined,
+    locale: undefined,
```

Comment:
undefined?

Reply:
yes.. it has to be undefined because, a default value implies that prop will always bypass the value from context... for example.. in this escenarios, if i set a default value:  
- context: undefined, component (default): en-US--- will use.. en-US --OK
- context: undefined, component: es-ES--- will use.. es-ES --OK
- context: es-ES, component: fr-FR--- will use.. fr-FR --OK
- context: es-ES, component (default): en-US--- will use.. en-US --NOT-OK

how ever.. if we do not set default value....
- context: undefined, component (default): undefined -- will use.. en-US as defined in helpers as default --OK
- context: undefined, component: es-ES--- will use.. es-ES --OK
- context: es-ES, component: fr-FR--- will use.. fr-FR --OK
- context: es-ES, component (default): undefined --- will use.. es-ES --NOT-OK